packages/x-codemod/src/util/addComponentsSlots.js

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
Was there a specific reason you chose to author this file as `js` instead of `ts`? 🤔

Reply:
Too much TS to fix, because I only consider nested objects.

Reply:
Mhh, fair point. 👌